components/feature/search/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <color name="mozac_feature_search_widget_background_color">@color/photonLightGrey10</color>
+    <color name="mozac_feature_search_widget_text_color_primary">@color/photonDarkGrey90</color>
+    <color name="mozac_feature_search_widget_color_primary">@color/photonDarkGrey90</color>
```

Comment:
I didn't see this being used anywhere on a quick search. Is this needed? I also saw we were search widget voice have a transparent background.

Reply:
Yes they are user in mozac_search_widget_large.xml and mozac_search_widget_medium.xml . I added background transparent for Microphone button because will appear with grey background if I don't do this.